playbooks/prepare_host.yaml

```
@@ -121,15 +120,14 @@
   # resolv.conf
   - name: Enable NetworkManager ownership of resolv.conf
     when:
-      - false
       - ansible_facts.distribution == 'CentOS'
     block:
       - name: Set main/dns=default in NetworkManager
```

Perhaps we should update the task name too?
```suggestion
      - name: Set main/dns in NetworkManager
```

---

Hello @mandre,
I test dev-install with the branch @EmilienM created for ibm baremetal instance. I have probably some others stuffs to add.
Thanks for the review